### ISSUE TITLE
Failure of wf 140.55 after merging PR 26485

### ISSUE THREAD

**Issue by fabiocos** [cms-sw/cmssw#26562]
After the integration of #26485 a failure is observed in the HI test workflow 140.55 step4:

```
----- Begin Fatal Exception 29-Apr-2019 13:33:41 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 263400 lumi: 65 event: 1020254 stream: 0
   [1] Prefetching for module HTMonitor/'AK8PFHT750_TrimMass50_Softdropmonitoring'
   [2] Prefetching for module GEDGsfElectronFinalizer/'gedGsfElectrons'
   [3] Prefetching for module PFProducer/'particleFlowTmp'
   [4] Prefetching for module ElectronIdMVABased/'mvaElectrons'
   [5] Prefetching for module GsfElectronSelector/'electronsWithPresel'
   [6] Calling method for module GsfElectronEcalDrivenProducer/'ecalDrivenGsfElectrons'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: std::vector<reco::Conversion>
Looking for module label: allConversions
Looking for productInstanceName: 

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```

**Comment by fabiocos** [comment]
assign reconstruction

**Comment by fabiocos** [comment]
@slava77 @perrotta @Sam-Harper apparently the update of the Egamma code breaks the old HI reconstruction sequence (140.55), as the required process.allConversions module is not run, why the 2018 HI sequence, that is basically the default pp one, is ok (140.56).

**Comment by Sam-Harper** [comment]
looking at it

**Comment by Sam-Harper** [comment]
okay the solution will be simply to have a flag whether this variable is to be filled or not and set it false for HION. Not elegant but no other better solution. Code is written, fixes workflow, testing other workflows.

**Comment by perrotta** [comment]
+1
- Fixed with #26567,  workflow 140.55 now passes
